doc/api/process.md

```diff
@@ -432,9 +432,11 @@ added: v0.5.0
 
 * {string}
 
-The `process.arch` property returns a String identifying the processor
-architecture that the Node.js process is currently running on. For instance
-`'arm'`, `'ia32'`, or `'x64'`.
+The `process.arch` property returns a string identifying the operating system CPU
+architecture *for which the Node.js binary was compiled*.
```

Comment:
The emphasis seems a bit too much but I see os.md does that too.

Reply:
I would lean towards removing the emphasis, it doesn't really add much. I'm sure @Trott would also have an opinion on this... 😉

Reply:
SGTM